scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -756,14 +756,16 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-secondary">Pa
   {% endmacro %}
 
 {% macro dismissed_gene_list(variant) %}
-  {% if variant.genes|length > 3%}
-    {{ variant.genes|length }}&nbsp;genes
-  {% else %}
-    {% for gene in variant.genes %}
-     <a href="http://www.genenames.org/cgi-bin/gene_symbol_report?match={{ gene.hgnc_symbol }}" target="_blank">{{ gene.hgnc_symbol }}&nbsp;<span class="badge badge-secondary">{{gene.region_annotation}}</span></a><br>
-    {% else %} <!-- 0 genes. strs for instance -->
-      <span class="badge badge-secondary">-</span>
-    {% endfor %}
+  {% if variant.genes %}
+    {% if variant.genes|length > 3%}
+      {{ variant.genes|length }}&nbsp;genes
+    {% else %}
+      {% for gene in variant.genes %}
+       <a href="http://www.genenames.org/cgi-bin/gene_symbol_report?match={{ gene.hgnc_symbol }}" target="_blank">{{ gene.hgnc_symbol }}&nbsp;<span class="badge badge-secondary">{{gene.region_annotation}}</span></a><br>
+      {% else %} <!-- 0 genes. strs for instance -->
```

Comment:
Hm, didn't you want that for the If variant.genes clause?

Reply:
Right, I'll fix, I just saw that `{% for gene in variant.genes %}` crashed, did't go much further down in the page!